src/celeritas/em/distribution/EnergyLossUrbanDistribution.hh

```diff
@@ -350,7 +350,7 @@ EnergyLossUrbanDistribution::sample_ionization_loss(Engine& rng)
         // which the energy loss is sampled for each collision (Eq. 20)
         UniformRealDistribution<real_type> sample_fraction(
             alpha / energy_ratio, 1);
-        for (CELER_MAYBE_UNUSED int i : range(n))
+        for ([[maybe_unused]] int i : range(n))
```

Comment:
That's weird-ish.  'i' is used implicit in the range checking (i guess).  Does the error message give any hint of a better pattern for this case?

Reply:
It's just for iterating over: the weirdness is that we have to declare a variable that takes the result of the implicit `*iter`, but then we don't use it. The only other way to get around it is to avoid using `range`.